website/search/share_search.py

```diff
@@ -220,3 +228,46 @@ def data_for_charts(elastic_results):
     }
 
     return all_data
+
+
+def to_atom(result):
+    return {
+        'title': result.get('title', 'No title provided'),
```

Comment:
`title` will still be defined as `''` or `None`.  Use `or` here for the expected behavior.